codeclimate-wrapper

```diff
@@ -41,17 +41,34 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+CC_XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config/}
```

Comment:
We shouldn't make up `CC_` versions of the XDG variables. We use them directly and default them if missing.

```sh
: "${XDG_CONFIG_HOME:=$HOME/.config}"
: "${XDG_CACHE_HOME:=$HOME/.cache}"
```